amcl/src/amcl_node.cpp

```diff
@@ -1436,8 +1443,32 @@ AmclNode::initialPoseReceived(const geometry_msgs::PoseWithCovarianceStampedCons
 }
 
 void
+AmclNode::initialPosesReceived(const move_base_msgs::PoseWithCovarianceStampedArrayConstPtr& msg)
+{
+  handleInitialPosesMessage(*msg);
+
+  // Force one no motion update
+  m_force_update = true;
```

Comment:
It is not a new variable but I agree that it should be changed to follow coding style.